index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Product landing page</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+
+    <header id="header">
+      <nav id="nav-bar">
+          <img id="header-img" class="logo" src="/samsung.png" alt="samsung-logo">
+          <ul class="nav-menu">
+            <li class="nav-link">
+              <a href="#video" target="_blank">Samsung phone</a>
+            </li>
+            <li class="nav-link">
+              <a href="http://samsung." target="_blank">Samsung tablets</a>
+            </li>
+            <li class="nav-link">
+              <a href="http://samsung." target="_blank">Samsung house</a>
+            </li>
+          </ul>
+      </nav>
+    </header>
+
+    <div class="video">
```

Comment:
- [ ] Again kindly consider adding `why` you are asking for this change. you can edit this comment and add at the end `Because using semantic tags like main would give your webpage a better score for semantics and accessibility`